admin/index.php

```diff
@@ -1208,13 +1208,13 @@ function get_support() {
 						<td><input required="required" type="text" class="bsf_text_medium" name="post_url" /></td></tr>
 					<tr><td><label for="subject"><strong>' . esc_html__( 'Subject:', 'all-in-one-schemaorg-rich-snippets' ) . '</strong> </label></td>
 						<td>
-							<select class="select_full" name="subject"> 
+							<select class="select_full" name="subject">
 								<option value="question"> I have a question </option>
 								<option value="bug"> I found a bug </option>
 								<option value="help"> I need help </option>
 								<option value="professional">  I need professional service </option>
 								<option value="contribute"> I want to contribute my code</option>
-								<option value="other">  Other </option>								
+								<option value="other">  Other </option>
```

Comment:
Can we make all these strings translatable @hemant7405 / @rohitp-bsf ?
I will approve this for now. Do add a task regarding it.